### ISSUE TITLE
Ensure Lume works in all major browsers (Chrome, Edge, Firefox, Safari)

### ISSUE THREAD

**Issue by trusktr** [lume/lume#124]
Some things were polyfilled automatically by Babel, but Buble does not polyfill anything, Buble is strictly a language transpiler.

We can test in all browsers and see what doesn't work, then include only the needed polyfills.

Babel is also currently giving us an error:

```
Uncaught ReferenceError: weights$1 is not defined
    at MutationObserver.eval (Utility.js?a470:56)
```

**Comment by trusktr** [comment]
The `weights$1` error was fixed in b985ec7bf8a1f1f8ace2d4051d7d522f9a1ce988. Now we just have to make sure all browser features are polyfilled for browsers that don't have them.

**Comment by trusktr** [comment]
We have since simplified code output and code itself, and browsers have advanced. Examples work everywhere now.
